.circleci/create_circleci_config.py

```diff
@@ -385,7 +385,10 @@ def create_circleci_config(folder=None):
 
     if len(jobs) > 0:
         config = {"version": "2.1"}
-        config["parameters"] = {"tests_to_run": {"type": "string", "default": test_list}}
+        config["parameters"] = {
```

Comment:
Might be good to have a comment `# Only used to accept the parameters from the trigger`